lib/src/pagination/pagination.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, Input, OnChanges, SimpleChanges, Output, EventEmitter, ViewEncapsulation } from "@angular/core";
+
+@Component({
+    selector: "sebng-pagination",
+    templateUrl: "./pagination.component.html",
+    styleUrls: ["./pagination.component.scss"],
+    encapsulation: ViewEncapsulation.None,
+})
+export class PaginationComponent implements OnChanges {
+    @Input() size: number;
+    @Output() handleChange: EventEmitter<any> = new EventEmitter();
```

Comment:
you don't need this, you already have `handleOnChange`, just implement [ControlValueAccessor](https://angular.io/api/forms/ControlValueAccessor) which will enable `value` input and `change` emitter

Reply:
I don't get it, its simple onChange, i simply rename to  handleChange. If any thing, i think the type of EventEmitter is wrong. Also why will you need ControlValueAccessor on a non input or controlled component ? I deviate from using change because it seems to be a reserve angular bind word and might be confusing.

Reply:
HTML doesn' have a full arsenal of customizable components, that's why we create **custom** ones. It's up to us to decide how it should behave. In this case, pagination has a value, and when it changes we need to reflect the change to the parent. It kind of behaves like a slider with a range of values. A slider is a form element. Therefore, we decided to make pagination behaves like a form element. With a `value` and `onChange` handler.

It is important to standardize implementations like this so that in the future we have standardized change handlers based on the `name` attribute.

Reply:
I'm expecting the component to be used like this:
```html
<sebng-pagination
    offset="5"
    size="25"
    [value]="value"
    (change)="handleChanges($event)
></sebng-pagination>
```

Reply:
We can still that. I rename it .